interface/sc-web-extensions/ui_components/plain-text/sc-web/components/plain_text/src/plain_text-paintPanel.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Paint panel.
+ */
+
+Example.PaintPanel = function (containerId) {
+	this.containerId = containerId;
+};
+
+Example.PaintPanel.prototype = {
+
+	init: function () {
+		this._callPlainTextRepresentationAgent(this.containerId);
+	},
+	
+	/* Call agent of searching semantic neighborhood,
+	send ui_main_menu node as parameter and add it in web window history
+	*/
+	_callPlainTextRepresentationAgent: function (containerId) {
+		var container = $('#' + containerId);
+		var self = this;
+		//Resolve sc-addr of current language
+		var lang = $('#language-select').find(":selected")[0].attributes['sc_addr'].value;
+		// Resolve sc-addr of question
+		var question = $('#history-items a').first().attr("sc_addr");
```

Comment:
```suggestion
		var question = $('a.history-item.active').attr("sc_addr");
``` 
would be better as we will get selected question in history

Reply:
Thanks